engine/runtime/src/main/scala/org/enso/interpreter/instrument/execution/PendingEdits.scala

```diff
@@ -11,12 +11,12 @@ trait PendingEdits {
     * @param file the edited file
     * @param edits the list of file edits
     */
-  def enqueue(file: File, edits: Seq[TextEdit]): Unit
+  def enqueue(file: File, edits: Seq[PendingEdit]): Unit
```

Comment:
This method would be `def enqueue(file: File, edits: Seq[TextEdit], execute: Boolean)`.